### PR TITLE
refactor: Synchronize versions with Endo

### PR DESCRIPTION
**Pull request by kriskowal**
This syncs versions with the latest published by the Endo repository. This requires a number of compatibility fixes.

This is the first in a sequence of changes toward achieving endo bundles #2684 with satisfactory performance https://github.com/endojs/endo/issues/655.

### CODE REVIEW

**Review by dckc — COMMENTED** [review]
I happened to take a look. So far, so good.

The static module record imports... maybe mark them with `// TODO(#nnn) ...`?

changes to yarn.lock are fairly modest; not many transitive dependencies

File: packages/xsnap/lib/lockdown-shim.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // @ts-nocheck
-import 'ses/lockdown';
+import 'ses';
```
Comment (dckc) [inline packages/xsnap/lib/lockdown-shim.js:2]: so this somehow no longer brings in babel?

File: packages/treasury/test/swingsetTests/test-treasury.js [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 // @ts-check
 
+// TODO Remove babel-standalone preinitialization
+// https://github.com/endojs/endo/issues/768
+import '@agoric/babel-standalone';
```
Comment (dckc) [inline packages/treasury/test/swingsetTests/test-treasury.js:7]: I suspect this should be using `prepare-test-env-ava.js`. @Chris-Hibbert ? @katelynsills ?

File: packages/zoe/test/swingsetTests/makeKind/test-makeKind.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
 /* global __dirname */
 
+// TODO Remove babel-standalone preinitialization
+// https://github.com/endojs/endo/issues/768
+import '@agoric/babel-standalone';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import '@agoric/install-metering-and-ses';
```
Comment (dckc) [inline packages/zoe/test/swingsetTests/makeKind/test-makeKind.js:7]: is there a version of `prepare-test-env-ava.js` with `metering`? @katelynsills ? @michaelfig ?

File: packages/xsnap/lib/lockdown-shim.js (hunk 1 above)
Comment (kriskowal) [reply packages/xsnap/lib/lockdown-shim.js:2]: Yes. I factored `@endo/static-module-record` out of SES, so SES no longer depends on Babel. StaticModuleRecord, however, _does_, so if you want it, you have to buy it. Compartment Mapper is refactored so you don’t need StaticModuleRecord to import `@endo/compartment-mapper/import-archive.js`, which is the subset we need in `@agoric/import-bundle`. So, you only need SMR if you’re building Zip bundles, not when you run them.

Comment (dckc) [reply packages/xsnap/lib/lockdown-shim.js:2]: Please promote this explanation to a commit message, if not a code comment (is there a relevant NEWS.md?)

That is: please hoist this valuable answer to "why bother?".

In fact, perhaps the PR title should be:
 - Integrate SES 0.13, which no longer depends on Babel

**Review by dckc — APPROVED** [review]
Nice work.

I think I understand this well enough that I could (help?) maintain it.

I suggest that the PR title should be more than administrative; something like:

 - Integrate SES 0.13, which no longer depends on Babel

Is there a relevant NEWS.md?

That is: please hoist this valuable answer to "why bother?".

Also, please hoist this bit from the (ephemeral/centralized) github PR comment to the decentralized history:
> This is the first in a sequence of changes toward achieving endo bundles #2684 with satisfactory performance endojs/endo#655.

File: packages/treasury/test/swingsetTests/test-treasury.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/treasury/test/swingsetTests/test-treasury.js:7]: That sounds right, but I don't see any consistency in other swingset tests to be able to tell what the model should be. 

Ah, I see `prepare-test-env-ava.js` is  a replacement for several lines here that setup `rawtest`, `test`, etc.

**Comment by kriskowal** [comment]
I’ve captured your suggested title and added significant background to the commit message. There should be no user-facing news since this is an internal refactor, except perhaps the need to pre-initialize Babel in deploy scripts.

**Pull request merged**